internal/util/util.go

```diff
@@ -178,8 +178,8 @@ func GetVolumeSnapshotContentForVolumeSnapshot(volSnap *snapshotv1api.VolumeSnap
 		// we need to wait for the VolumeSnaphotContent to have a snapshot handle because during restore,
 		// we'll use that snapshot handle as the source for the VolumeSnapshotContent so it's statically
 		// bound to the existing snapshot.
-		if snapshotContent.Status == nil || snapshotContent.Status.SnapshotHandle == nil {
-			log.Infof("Waiting for volumesnapshotcontents %s to have snapshot handle. Retrying in %ds", snapshotContent.Name, interval/time.Second)
+		if snapshotContent.Status == nil || snapshotContent.Status.SnapshotHandle == nil || snapshotContent.Status.ReadyToUse == nil || !*(snapshotContent.Status.ReadyToUse) {
```

Comment:
Could you use
https://github.com/vmware-tanzu/velero/blob/a111eed2af8c0bb37ecd6d682b3faad116fbad2f/pkg/util/boolptr/boolptr.go#L20

Reply:
Right, there is already a tool function to do that.